overlord/overlord.go

```diff
@@ -347,6 +346,29 @@ func (o *Overlord) Settle(timeout time.Duration) error {
 	return nil
 }
 
+// Settle runs first a state engine Ensure and then wait for
+// activities to settle. That's done by waiting for all managers
+// activities to settle while making sure no immediate further Ensure
+// is scheduled. It then waits similarly for all ready changes to reach
+// the clean state. Chiefly for tests. Cannot be used in conjunction
```

Comment:
a clean state? or is there one definitive clean state?

Reply:
it's in the sense of Change.IsClean